eisbuk-admin/src/translations/it.json

```diff
@@ -60,7 +60,9 @@
     "InvalidTime": "Formato dell'ora non valido (\"HH:mm\")",
     "InvalidDate": "Formato della data non valido (\"dd/mm/yyyy\")",
     "InvalidPhone": "Inserire un numero di telefono valido, completo di prefisso internazionale (+39 per l'Italia)",
-    "TimeMismatch": "L'ora di inizio è maggiore dell'ora di fine"
+    "TimeMismatch": "L'ora di inizio è maggiore dell'ora di fine",
+    "InvalidSmsFrom": "Formato non valido: Solo lettere o numeri e nessun simbolo/spazio",
```

Comment:
i don't get this one

Reply:
Sms sender doesn't need to be a number (it can be `IgorIce`) for instance. Like...when I get a 2FA SMS from Binance, It says from: `Binance` (not the phone number)